### ISSUE TITLE
pylint under pipenv: [E0611(no-name-in-module), ] No name 'CNAME' in module 'dns.rdatatype'

### ISSUE THREAD

**Issue by e2m4n** [PyCQA/pylint#3935]
Importing constants from `dns.rdatatype` gives error E0611 when running pylint under pipenv.

Pylint running under virtualenv doesn't give the error.

### Prerequisites
1. Have a virtualenv and a pipenv for testing.
2. Install the module `dnspython` from pypi in both environments.
3. Install pylint in both environments.

### Steps to reproduce
1. Create test python script: `echo "from dns.rdatatype import CNAME" > /tmp/pylinttest.py`
2. Run pylint on test script with pipenv activated.
3. Run pylint on test script with virtualenv activated.

### Current behavior

Current behaviour is this output from pipenv:
```
# pylint /tmp/pylinttest.py
************* Module pylinttest
/tmp/pylinttest.py:1: [C0114(missing-module-docstring), ] Missing module docstring
/tmp/pylinttest.py:1: [E0611(no-name-in-module), ] No name 'CNAME' in module 'dns.rdatatype'
/tmp/pylinttest.py:1: [W0611(unused-import), ] Unused CNAME imported from dns.rdatatype
```

### Expected behavior

Expected behaviour is this output from virtualenv:
```
# pylint /tmp/pylinttest.py
************* Module pylinttest
/tmp/pylinttest.py:1:0: C0114: Missing module docstring (missing-module-docstring)
/tmp/pylinttest.py:1:0: W0611: Unused CNAME imported from dns.rdatatype (unused-import)
```

### pylint --version output
```
pylint 2.6.0
astroid 2.4.2
Python 3.6.12 (default, Sep 21 2020, 18:34:10) 
[GCC 10.2.0]
```

### Additional notes

Please note that under both virtual environments the module `dns.rdatatype` can be imported without problem. The error is only with pylint.

By the way, the way dnspython exposes its constants is by calling `update()` on its `globals()`:
```
globals().update(RdataType.__members__)
```
(rdatatype.py line 118)

**Comment by e2m4n** [comment]
Spending more time researching this, it appears that this problem is something introduced by dnspython in its last version - v2.0 - and is fixed in its next version - v2.1
Will close this bug now.
